### PR TITLE
Fix linguistic errors

### PR DESCRIPTION
**Pull request by GiorgiBeriashvili**
An attempt to improve the reading experience of the documents by fixing linguistic errors.

### CODE REVIEW

File: linear_algebra-1.ipynb [hunk 1]
```diff
@@ -294,7 +294,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Every subset $R$ of $AxB$ is called relation. We write $aRb$ iff $(a, b) \\in R$"
+    "Every subset $R$ of $AxB$ is called relation. We write $aRb$ if $(a, b) \\in R$"
```
Comment (sosoxuc) [inline linear_algebra-1.ipynb]: Maybe @levants meant here `if and only if`?

**Comment by sosoxuc** [comment]
@GiorgiBeriashvili thanks for being on top of this, assigned to Levani.

File: linear_algebra-1.ipynb (hunk 1 above)
Comment (sosoxuc) [reply linear_algebra-1.ipynb]: @GiorgiBeriashvili consulted w/ @levants 
`iff` means `if and only if`, can you please revert this line?

Comment (GiorgiBeriashvili) [reply linear_algebra-1.ipynb]: @sosoxuc

Of course! I'm on it.

**EDIT:** Done! I will resolve this conversation.

**Pull request merged**